doc/releases/changelog-dev.md

```diff
@@ -417,6 +417,7 @@
 * The JAX-JIT interface now supports gradient transforms and device gradient execution in `backward` mode with the new
   return types system.
   [(#3235)](https://github.com/PennyLaneAI/pennylane/pull/3235)
+  [(#3445)](https://github.com/PennyLaneAI/pennylane/pull/3445)
```

Comment:
I think this changelog entry should mention that forward mode is also supported now.

Reply:
Good catch! I've removed the specifics mentioned as now all the cases other interfaces support are also supported here.